scripts/docker-entrypoint-alice.sh

```diff
@@ -11,7 +11,7 @@ echo "Chain Version is ${CHAIN_VERSION}"
   --rpc-cors=all \
   --base-path /tmp/polkadot-chains/alice \
   --keystore-path "/tmp/polkadot-chains/alice/keys" \
-  --chain /dhx/node/node/src/chain-built/chain_def_${CHAIN_VERSION}.json \
+  --chain ${CHAIN_VERSION} \
```

Comment:
i can only see this change for Alice, Charlie, Dave, Eve.
so i think we need to do the same for Bob